### PR TITLE
All UI Changes are done

### PR DESCRIPTION
**Pull request by shivamsherje**
Ask llama logo takes loading because it was a link so saved it locally .
Clear chat button adjustments was not on place because chat box width is increase fixed it 
remove background on footer make it black color looks well than previous 
Adjust error prompt on the llama it was also not on place because chat box width is increase fixed it 
change the font-family of header footer and  disclaimer title and placeholder hope u like it 
After selecting the master ,  master name is getting bold now
 

**few changes was in previous commit and i merge it because small UI changes but necessary changes !!!**

### CODE REVIEW

File: src/components/Masters.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="masters">
-    <h4>Select Masters</h4>
+    <h4 style="font-family:'Trebuchet MS', sans-serif;">Select Masters</h4>
```
Comment (AishwaryShrivastav) [inline src/components/Masters.vue:3]: why are we adding fonts in HTML section and not adding them in CSS section?

File: src/components/Header.vue [hunk 2]
```diff
@@ -122,11 +123,15 @@ h2 {
 h1 {
   /* font-size: 2.5rem; */
   margin-bottom: 0.2rem;
+  font-family: 'Trebuchet MS', sans-serif;
 }
 
 h2 {
-  font-size: 1.2rem;
+  font-size: 1.1rem;
```
Comment (AishwaryShrivastav) [inline src/components/Header.vue:130]: Are we using these fonts only in some specific locations?

File: src/components/Footer.vue [hunk 3]
```diff
@@ -229,9 +229,10 @@ export default {
   /* height: 10vh; */
 }
 .new_footer {
+  font-family: "Trebuchet MS", sans-serif;
```
Comment (AishwaryShrivastav) [inline src/components/Footer.vue:232]: Why are'nt we adding font in main app?

**Review by AishwaryShrivastav — COMMENTED** [review]
Added few comment related to Fonts usage

File: src/components/Masters.vue (hunk 1 above)
Comment (shivamsherje) [reply src/components/Masters.vue:3]: ok will fix it !!

File: src/components/Header.vue (hunk 2 above)
Comment (shivamsherje) [reply src/components/Header.vue:130]: because sir said only change the heading fonts

File: src/components/Footer.vue (hunk 3 above)
Comment (shivamsherje) [reply src/components/Footer.vue:232]: can i make whole app  of  same font  it will also looks good

**Pull request merged**